RFS/src/main/java/com/rfs/transformers/Transformer_ES_6_8_to_OS_2_11.java

```diff
@@ -47,21 +56,26 @@ public ObjectNode transformGlobalMetadata(ObjectNode root) {
         componentTemplatesRoot.set("component_template", componentTemplatesSubRoot);
         newRoot.set("component_template", componentTemplatesRoot);
 
-        return newRoot;
+        return new GlobalMetadataData_OS_2_11(newRoot);
     }
 
     @Override
-    public ObjectNode transformIndexMetadata(ObjectNode root){
-        ObjectNode newRoot = root.deepCopy();
+    public IndexMetadata transformIndexMetadata(IndexMetadata index) {
+        var copy = index.deepCopy();
+        tranformIndex(copy);
+        return copy;
+    }
+
+    private void tranformIndex(Index index) {
+        logger.debug("Original Object: " + index.raw().toString());
+        var newRoot = index.raw();
 
-        TransformFunctions.removeIntermediateMappingsLevels(newRoot);
```

Comment:
Was this just not necessary?  Or are we calling it elsewhere?

Reply:
This is in that method AFAIK.  Unless you're calling it via the `indexTransformations.forEach()` call, which I don't think you are?

Reply:
Oh, I understand now.  It is being invoked in your list of transformations; I just didn't understand what the transform in there was doing.  The new comment in that transformation class clarified things.  Thanks!